src/main/java/org/opensearch/ad/transport/ValidateAnomalyDetectorTransportAction.java

```diff
@@ -243,11 +243,7 @@ private void checkIndicesAndExecute(
                 // parsed to a DetectorValidationIssue that is returned to
                 // the user as a response indicating index doesn't exist
                 DetectorValidationIssue issue = parseADValidationException(
-                    new ADValidationException(
-                        ADCommonMessages.INDEX_NOT_FOUND,
-                        DetectorValidationIssueType.INDICES,
-                        ValidationAspect.DETECTOR
-                    )
+                    new ValidationException(ADCommonMessages.INDEX_NOT_FOUND, ValidationIssueType.INDICES, ValidationAspect.DETECTOR)
```

Comment:
would spotless complain about the formatting later?

Reply:
seems like the change is from spotless :)